odoo/models.py

```diff
@@ -2246,11 +2246,16 @@ def _read_group_prepare(self, orderby, aggregated_fields, annotated_groupbys, qu
             for key in ('field', 'groupby')
         }
         for order_part in orderby.split(','):
-            order_split = order_part.split()
+            order_split = order_part.split()  # potentially ["field:agg", "desc"]
```

Comment:
This is technically not an _aggregate_, but a _projection_.  So I suggest mentioning `"field:func"` or `"field.proj"` instead.

Reply:
while true, projection in this context is less clear than aggregation IMHO. I went for group_func to reuse the terms of the docstring of read_group